client/hosting/performance/components/PageSelector.tsx

```diff
@@ -16,12 +16,24 @@ export const PageSelector = ( props: ComponentProps< typeof SearchableDropdown >
 				<SearchableDropdown
 					{ ...props }
 					className="site-performance__page-selector-drowdown"
-					__experimentalRenderItem={ ( { item } ) => (
-						<div className="site-performance__page-selector-item" aria-label={ item.label }>
-							<span>{ item.label }</span>
-							<span className="subtitle">{ item.path }</span>
-						</div>
-					) }
+					__experimentalRenderItem={ ( { item } ) => {
+						if ( item.value === '-1' ) {
+							return (
+								<div className="message">
+									{ translate( 'Searching your top 20 most popular pages.' ) }
```

Comment:
"Searching your top 20 most popular pages" may give a feeling that it's still retrieving the pages since only ten are shown in the dropdown.
<img width="307" alt="image" src="https://github.com/user-attachments/assets/66ce95be-5831-4f37-9aa7-c399eaa413df">

Wondering if these would be a clearer copy:
* Search is limited to the top 20 pages.
* Top 20 pages available for search.

Reply:
@matt-west curious what you think of the copy from a UX point of view 🙏

Reply:
Thanks for highlighting @candy02058912. I agree that we can improve on the current copy.

We're really talking about a limitation of the performance feature here rather than just the search.

Something like this could work, but it's a bit long.

> Performance testing is available for the 20 most popular pages.

Reply:
cc: @autumnfjeld what's your take on the copy suggestions here?

Reply:
@candy02058912  

> Performance testing is available for the 20 most popular pages.

If this fits on two lines, this is good.

Slightly shorter option if needed and if @matt-west  approved

> Performance tests limited to the 20 most popular pages.


> Performance tests limited to the top 20 pages.